conda_smithy/configure_feedstock.py

```diff
@@ -1134,6 +1134,8 @@ def render_README(jinja_env, forge_config, forge_dir):
         trim_skip=False,
     )
 
+    if "README.md" in forge_config["skip_render"]:
+        return
```

Comment:
Can you move this to the top of the function? We can avoid `conda_build.api.render` in this case.

Reply:
You mean move the if check to main()? something like
```python
def main():
    render_others()
    if :
       render_readme()
```

Reply:
No, move the two line above to Line 1127

Reply:
> Can you move this to the top of the function? We can avoid `conda_build.api.render` in this case.

Ahh, sorry, I see your point. I overlooked the above line which I thought they were part of the arguments...